x-pack/plugins/search_playground/public/hooks/use_usage_tracker.ts

```diff
@@ -14,24 +14,25 @@ const APP_TRACKER_NAME = 'search_playground';
 export const useUsageTracker = () => {
   const { usageCollection } = useKibana().services;
 
-  return useMemo(
-    () => ({
-      click: usageCollection.reportUiCounter.bind(
-        usageCollection,
-        APP_TRACKER_NAME,
-        METRIC_TYPE.CLICK
-      ),
-      count: usageCollection.reportUiCounter.bind(
-        usageCollection,
-        APP_TRACKER_NAME,
-        METRIC_TYPE.COUNT
-      ),
-      load: usageCollection.reportUiCounter.bind(
-        usageCollection,
-        APP_TRACKER_NAME,
-        METRIC_TYPE.LOADED
-      ),
-    }),
-    [usageCollection]
-  );
+  return useMemo(() => {
+    if (usageCollection) {
+      return {
+        click: usageCollection.reportUiCounter.bind(
+          usageCollection,
+          APP_TRACKER_NAME,
+          METRIC_TYPE.CLICK
+        ),
+        count: usageCollection.reportUiCounter.bind(
+          usageCollection,
+          APP_TRACKER_NAME,
+          METRIC_TYPE.COUNT
+        ),
+        load: usageCollection.reportUiCounter.bind(
+          usageCollection,
+          APP_TRACKER_NAME,
+          METRIC_TYPE.LOADED
+        ),
+      };
+    }
```

Comment:
If we return an empty object, we get the same error as we are getting now. Because it looks for `.click`, or `.load` in the object. Thus leads to the same error.